src/movies.js

```diff
@@ -1,25 +1,103 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+   const getDirectors = movies.map((movie, index) => {
+   return movie.director
+})  
+     const uniqueArr = getDirectors.filter((movie, index) => {
+    return getDirectors.indexOf(movie) === index;
+  });
+  
+return uniqueArr;
+}
+
+
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies(moviesArray) {}
+function howManyMovies(moviesArray) {
+    const spilbergMovies = moviesArray.filter((movie) => movie.director === 'Steven Spielberg')
+    const dramaSpilbergMovies = spilbergMovies.filter((dramaMovies) => dramaMovies.genre.includes('Drama'))
```

Comment:
El filter te deja poner mas de una condicion de filtrado!, esta genial! pero es mejor si pones las dos condiciones en un unico filter :D
```suggestion
    const dramaSpilbergMovies = moviesArray.filter((movie) => movie.director === 'Steven Spielberg' && movie.genre.includes('Drama'))
```